### DISCUSSION TITLE
New Feature Request: Developer Interviews

### DISCUSSION THREAD

**Discussion by kera-cudmore** [codu-code/codu#578] [Ideas]
## Context

My boss was recently interviewed for a blog where they interview different developers each week/month about their developer job, how they got started, what tech they use, what they're currently learning/excited about, book recs etc.

Thought this could potentially be a cool feature to add to the site, especially as we have such a wide range of developers at all stages using lots of different tech

**Comment by NiallJoeMaher** [comment]
Are you thinking a series or a weekly/monthly feature on developers answering questions?
A podcast or text format? 

Really fun idea tbh!

**Comment by kera-cudmore** [comment]
Hmmm... could be either/both to be honest!

I know that I'm always interested to know what other developers do for their jobs - but also what they're spending their free time learning/doing too

**Comment by s0ham075** [comment]
What if we show the skill or technology the user is learning on user profile page?? Similar thing could be done for the what they do at their job. Also we can create a timeline component which shows the previous such technologies the user was learning , essentially creating a roadmap of their journey, which we can customize to add the book, resources etc which they used to learn that particular technology

**Comment by NiallJoeMaher** [comment]
@kera-cudmore I'm thinking a podcast for some extra content 😉 
![codercast](https://github.com/codu-code/codu/assets/12615742/a2bf1509-3dad-474e-ab18-0f7b0f000e2a)

@s0ham075 I think it could be useful to be able to tag people's profiles with certain technologies eg someone could pick "AWS", "NODE", "JAVASCRIPT". Might help if we want to try and help people get job placements in the future too

**Comment by anyahush** [comment]
I think this is an awesome idea! I'm always curious as to what other devs do in their day-to-day jobs. Would be keen to help/contribute on this one :)

**Comment by rohan057** [comment]
Hi guys we have a Graduate Programme for 2024 if interested read below and apply.

Graduate Programme 2024: Software Development

About Us:
Accolite Digital, a leading digital transformation services provider, is excited to announce the launch of its Software Development Graduate Programme in 2024.

Position: Software Developer - Python/Java

Location: Glasgow

Programme Start: January 2024

Your Role:
As a Software Developer with Accolite Digital, you will have the opportunity to work on-site, starting in January 2024. Your initial engagement will be with a Fortune 100 global investment bank in the financial services industry. Responsibilities include designing, developing, modifying, and maintaining software products.

What We Offer:

Comprehensive development plan
Extensive and continuous training in banking, financial services, and relevant technologies
Exposure to cutting-edge technology and a dynamic community of technologists
Requirements:

MSc/BSc in a technical-related degree (Computer Science, Computing-Related Fields)
Graduated between 2019-2022
Genuine interest in technology and programming
Coding proficiency in Java and/or Python
Strong understanding of object-oriented concepts
Attention to detail, quality-focused, and adherence to standards/best practices
Ability to thrive in a fast-paced environment
Excellent written and verbal communication skills
About Accolite Digital:
Accolite Digital is an innovative digital transformation services provider, applying design thinking to solve complex business problems for Fortune 500 companies. We believe in a bias-free world and encourage diversity in our workforce.

Why Join Us:
At Accolite Digital, we believe a bias-free world is an empowered world. We welcome applications from everyone, especially women and individuals from diverse backgrounds, to contribute to building the next generation of Accolite Digital.

Ready to Launch Your Career in Software Development?
Apply now for the Software Development Graduate Programme and be part of a community of bright and passionate technologists shaping the future at Accolite Digital.

Python Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37
Java Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37

Reply back if you have any further questions and I will be happy to answer, else best of luck with your applications.
